### PR TITLE
psycho mantis?

### PR DESCRIPTION
**Pull request by Iamgoofball**
hrm

### CODE REVIEW

**Comment by Jxhnathan08** [comment]
Writing will be reviewed before merging, but this seems like a great idea. 

@monikLover

**Comment by Iamgoofball** [comment]
maintainer edits are enabled so do whatever

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -2777,6 +2777,21 @@ label monika_eternity:
...
     return
 
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['metal gear' , 'wolbachia' , 'kojima' , 'konami']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_mgs') #id
+    monika_random_topics.append('monika_sleep')
+
+label monika_sleep:
```
Comment (monikLover) [inline Monika After Story/game/script-topics.rpy]: Topic is fine. Might want to change the reference to something that wouldn't involve the game canon-wise i.e. scrapped designs, possible hidden stuff.

**Comment by Iamgoofball** [comment]
>reverts "psycho mantis"

why did you merge it then?

**Comment by Jxhnathan08** [comment]
Looking into this.

**Comment by therationalpi** [comment]
Accidental because this was committed to master instead of content.

**Comment by Iamgoofball** [comment]
Nice

**Pull request merged**